RichEditorView/Assets/editor/rich_editor.js

```diff
@@ -356,3 +356,27 @@ RE.getSelectedHref = function() {
 
     return href ? href : null;
 };
+
+/* retrieve caret vertical position */
+
+RE.getCaretPosition = function() {
+    var x=0, y=0;
+    var sel=window.getSelection();
+    if (sel.rangeCount) {
+        var range = sel.getRangeAt(0);
+        var needsWorkAround = (range.startContainer.nodeName.toLowerCase() == 'div' && range.startOffset == 0);
```

Comment:
Could you add a comment or something about what this workaround is for and what it's doing?